sources/Valkey.Glide/Abstract/ValkeyServer.cs

```diff
@@ -89,4 +89,16 @@ public async Task<ValkeyValue> EchoAsync(ValkeyValue message, CommandFlags flags
         Utils.Requires<NotImplementedException>(flags == CommandFlags.None, "Command flags are not supported by GLIDE");
         return await _conn.Command(Request.Echo(message), new ByAddressRoute(EndPoint.ToString()!));
     }
+
+    public async Task<ValkeyValue> ClientGetNameAsync(CommandFlags flags = CommandFlags.None)
+    {
+        Utils.Requires<NotImplementedException>(flags == CommandFlags.None, "Command flags are not supported by GLIDE");
+        return await _conn.Command(Request.ClientGetName(), new ByAddressRoute(EndPoint.ToString()!));
+    }
+
+    public async Task<long> ClientIdAsync(CommandFlags flags = CommandFlags.None)
+    {
+        Utils.Requires<NotImplementedException>(flags == CommandFlags.None, "Command flags are not supported by GLIDE");
+        return await _conn.Command(Request.ClientId(), new ByAddressRoute(EndPoint.ToString()!));
```

Comment:
```suggestion
        return await _conn.Command(Request.ClientId(), MakeRoute());
```
Update existing APIs too